packages/sitecore-jss/src/layout-service.ts

```diff
@@ -58,6 +59,17 @@ export type RestLayoutServiceConfig = {
   dataFetcherResolver?: DataFetcherResolver;
 };
 
+export type GraphQLLayoutServiceConfig = {
+  /**
+   * Your Graphql endpoint
+   */
+  apiHost: string;
+  /**
+   * The JSS application name
+   */
+  siteName: string;
+};
```

Comment:
Do we have to support the SSC API key (similar to REST layout service)? GraphQL endpoints support this, but perhaps the default edge endpoints don't have this security configured?

Reply:
@ambrauer In config.js we have graphqlEndpoint:
```
config.graphQLEndpointPath = process.env.GRAPH_QL_ENDPOINT_PATH || "/sitecore/api/graph/edge",
config.graphQLEndpoint = `${config.sitecoreApiHost}${config.graphQLEndpointPath}?sc_apikey=${config.sitecoreApiKey}`;
```
which includes sc_apikey

Reply:
Ahh, yes, forgot that was part of the computed value.